pkg/filesystem/subvolume.go

```diff
@@ -296,6 +348,19 @@ func unMarshaljson(list string) []fsStruct {
 	return unmarshal
 }
 
+// deleteSnapshot deletes the subvolume snapshot
+func deleteSnapshot(ctx context.Context, clientsets *k8sutil.Clientsets, OperatorNamespace, CephClusterNamespace, fs, subvol, svg, snap string) {
```

Comment:
parameters are generally `camelCase`
```suggestion
func deleteSnapshot(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace, fs, subvol, svg, snap string) {
```